src/web/components/StickyBottomBanner/ReaderRevenueBanner.tsx

```diff
@@ -97,26 +130,39 @@ const MemoisedInner = ({
                     return;
                 }
 
-                const { module } = json.data;
+                const { module, meta } = json.data;
 
                 import(/* webpackIgnore: true */ module.url)
                     .then(bannerModule => {
                         setBannerProps({
                             ...module.props,
                         });
                         setBanner(() => bannerModule.Banner); // useState requires functions to be wrapped
-                        sendOphanEvent();
+                        setBannerMeta(meta);
+                        sendOphanBannerEvent('INSERT', meta);
```

Comment:
`INSERT` could be a const or enum in a shared file as it's currently used here and in slotBody as a string. Same for `VIEW`.